.gitignore

```diff
@@ -105,3 +105,6 @@ dist
 
 .parcel-cache
 example/dev-bundle
+example/bundle
+.vscode/
+build/
```

Comment:
Excellent work! This looks great - last thing, though. Can we remove the `example/bundle` and ` build` directories from the gitignore? These are needed for the built examples branch. Perhaps once #150 is solved we can remove the need for a separate github pages branch but for the moment there should be no need to build the project during development.

Reply:
Got it! I didn't realize the examples were built and served using gh pages. I will update the gitignore right away.